[21/May/2025 14:00:57] "GET / HTTP/1.1" 200 13079
Internal Server Error: /documents/
Traceback (most recent call last):
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/base.py", line 875, in _resolve_lookup
    current = current[bit]
TypeError: 'FieldFile' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapper_view
    return view_func(request, *args, **kwargs)
  File "/home/runner/workspace/core/views/views.py", line 244, in documents
    return render(request, 'home/documents.html', {'documents': documents})
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/base.py", line 171, in render
    return self._render(context)
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/base.py", line 1000, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/loader_tags.py", line 159, in render
    return compiled_parent._render(context)
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/base.py", line 1000, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/base.py", line 1000, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/defaulttags.py", line 325, in render
    return nodelist.render(context)
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/base.py", line 1000, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/defaulttags.py", line 241, in render
    nodelist.append(node.render_annotated(context))
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/base.py", line 1059, in render
    output = self.filter_expression.resolve(context)
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/base.py", line 710, in resolve
    obj = self.var.resolve(context)
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/base.py", line 842, in resolve
    value = self._resolve_lookup(context)
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/template/base.py", line 885, in _resolve_lookup
    current = getattr(current, bit)
  File "/home/runner/workspace/.pythonlibs/lib/python3.10/site-packages/django/db/models/fields/files.py", line 67, in url
    return self.storage.url(self.name)
  File "/home/runner/workspace/core/storage.py", line 19, in url
    return self.client.get_url(name)
AttributeError: 'Client' object has no attribute 'get_url'
[21/May/2025 14:01:04] "GET /documents/ HTTP/1.1" 500 239962